app/models/imports/docx.rb

```diff
@@ -2,5 +2,28 @@ module Imports
   class Docx < Import
     has_one :pdf, as: :parent, dependent: :destroy, autosave: true
     has_one_attached :file
+
+    def process
+      output_pdf_path = ConvertDocToPdf.call(id, file)
+
+      transaction do
+        create_pdf!(
+          file: File.open(output_pdf_path),
+          public_document: public_document,
+          metadata: metadata,
+        )
+        update!(
+          processed_at: Time.now,
+          processing_errors: nil,
+          status: :completed,
+        )
+      end
+    rescue ConvertDocToPdf::PdfConversionError => e
+      update!(
+        processed_at: nil,
+        processing_errors: e.message,
+        status: :needs_support,
```

Comment:
Is this `status` value correct? Are any of these `status` values correct?

(They are meant to mimic the SourceFile status value.)

Reply:
@mike-burns Right now, I can't think of a reason to change them, but I'm sure once we go deeper, there may be some statuses that won't be valid for some imports. 

For this specific case, needs support seems correct. @littleforest can you help us with this one?

Reply:
The `needs_support` status is really to indicate that help is needed deciphering how data should get transferred from the doc/pdf file into the spreadsheet that is uploaded. So maybe we need a new status, something like `backend_support`, or `convert_failure`?